### PR TITLE
cleanup(libsinsp,libscap): update userspace/common include and strlcpy paths

### PR DESCRIPTION
**Pull request by LucaGuerra**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind cleanup

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area build

/area libsinsp

/area libscap

**Does this PR require a change in the driver versions?**

No

**What this PR does / why we need it**:

As discussed with @Molter73 , update include paths across the libs to be able to include `userspace/common` without ugly `../../` 

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/modern_bpf/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 message(STATUS "Modern BPF tests build enabled")
 
+include_directories(../userspace/common)
```
Comment (Molter73) [inline test/modern_bpf/CMakeLists.txt:3]: Could we use `${CMAKE_SOURCE_DIR}` to anchor this include a bit more?
```suggestion
include_directories(${CMAKE_SOURCE_DIR}/userspace/common)
```

**Review by Molter73 — APPROVED** [review]
/lgtm

Neat! Thanks for this Luca!

File: test/modern_bpf/CMakeLists.txt (hunk 1 above)
Comment (LucaGuerra) [reply test/modern_bpf/CMakeLists.txt:3]: Question is, would that work if we build Falco with an external `FALCOSECURITY_LIBS_SOURCE_DIR`?

Comment (Molter73) [reply test/modern_bpf/CMakeLists.txt:3]: Probably not 😅

**Review by Molter73 — COMMENTED** [review]
/lgtm

File: test/modern_bpf/CMakeLists.txt (hunk 1 above)
Comment (FedeDP) [reply test/modern_bpf/CMakeLists.txt:3]: Was this concern addressed? :)

Comment (Molter73) [reply test/modern_bpf/CMakeLists.txt:3]: I think so, we can keep the relative path for now.

**Review by Molter73 — APPROVED** [review]
/lgtm

**Review by FedeDP — DISMISSED** [review]
/approve

**Comment by LucaGuerra** [comment]
@Andreagit97 @FedeDP rebased

**Review by FedeDP — APPROVED** [review]
/approve

**Review by Andreagit97 — APPROVED** [review]
/approve
thank you!

**Pull request merged**